app/src/main/java/org/gdg/frisbee/android/pulse/PulseAdapter.java

```diff
@@ -39,13 +40,18 @@
 
     private LayoutInflater mInflater;
     private ArrayList<Map.Entry<String, PulseEntry>> mPulse;
-    private int[] mPosition;
+    private int[] mPositions;
```

Comment:
No :) While I was doing it I also refactored it. mPosition is not suitable because it was not plural. We are holding multiple items in it.